jest.integration.config.ts

```diff
@@ -4,7 +4,6 @@ const config: Config.InitialOptions = {
   coveragePathIgnorePatterns: ['/node_modules/', '/dist/'],
   testPathIgnorePatterns: ['/src/'],
   preset: 'ts-jest',
-  setupFiles: ['<rootDir>/.integration.test.env'],
```

Comment:
Can we put this back, just give it a default of build?

Reason being is you can also run the tests with `yarn jest ...` this means there is 1 less argument to add.

Reply:
As discussed, `yarn test:integration` can be used just like `yarn jest`.